apps/console/src/features/groups/components/group-common-components/render-chip.tsx

```diff
@@ -84,12 +86,15 @@ export const RenderChipRolesInGroups: FunctionComponent<RenderChipInterface> = (
             <Chip
                 { ...props }
                 key={ key }
-                label={ 
+                label={
                     (<>
-                        <i> { audienceType } </i> 
-                        <i> { audienceType === "application" && ( " : " + audienceDisplay ) } </i>
-                        { " | " }
-                        <strong> { displayName } </strong>
+                        <Typography fontWeight={ 500 } sx={ { display: "inline-block" } }>{ displayName }</Typography>
+                        <Typography sx={ { display: "inline-block", fontStyle:"italic" } } >
```

Comment:
Avoid using sx prop. You can create a style file specific to this component and place these styles there. Now we are following this approach. Please check other places as well.
Ex: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/console-settings/components/console-settings-tabs.scss